pkg/mgmt/lvmnode/builder.go

```diff
@@ -17,34 +17,46 @@
 package lvmnode
 
 import (
-	"time"
-
-	clientset "github.com/openebs/lvm-localpv/pkg/generated/clientset/internalclientset"
-	openebsScheme "github.com/openebs/lvm-localpv/pkg/generated/clientset/internalclientset/scheme"
-	informers "github.com/openebs/lvm-localpv/pkg/generated/informer/externalversions"
-	listers "github.com/openebs/lvm-localpv/pkg/generated/lister/lvm/v1alpha1"
 	corev1 "k8s.io/api/core/v1"
 	metav1 "k8s.io/apimachinery/pkg/apis/meta/v1"
+	"k8s.io/apimachinery/pkg/runtime/schema"
+	"k8s.io/client-go/dynamic"
+	"k8s.io/client-go/dynamic/dynamicinformer"
+	"k8s.io/client-go/dynamic/dynamiclister"
 	"k8s.io/client-go/kubernetes"
 	"k8s.io/client-go/kubernetes/scheme"
 	typedcorev1 "k8s.io/client-go/kubernetes/typed/core/v1"
 	"k8s.io/client-go/tools/cache"
 	"k8s.io/client-go/tools/record"
 	"k8s.io/client-go/util/workqueue"
 	"k8s.io/klog"
+	"time"
+)
+
+const (
+	controllerAgentName = "lvmnode-controller"
+	GroupOpenebsIO      = "local.openebs.io"
+	VersionV1alpha1     = "v1alpha1"
+	Resource            = "lvmnodes"
 )
 
-const controllerAgentName = "lvmnode-controller"
+var resource = schema.GroupVersionResource{
```

Comment:
Instead of just naming it as resource, can we rename it to `lnResource` where `ln` would represent `lvm node`. I have seen this kind of naming standard being followed in a lot of places for dynamic resource controllers.